app/models/rent.rb

```diff
@@ -138,11 +140,9 @@ def owner?(user)
   # /d.wessman
   def authorize(access)
     if ((access.present?) && (self.access_code.present?) && (self.access_code == access))
-      puts 'access true'
       return true
     else
-      puts 'access fails'
-      errors.add('Auktorisering', 'misslyckades, kontrollera ev. kod.')
+      errors.add("Auktorisering", "misslyckades, kontrollera ev. kod.")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.